Факториал/govnoscript.js

```diff
@@ -0,0 +1,13 @@
+ function fac(){
+	var c = document.getElementById('ch').value,C=c
+	if(c==1||c==0) {
```

Comment:
refactor: space after if, space before and after == and ||
bug: ==, use === instead